Test/PcapPacket.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using PacketDotNet;
+using System.IO;
+
+namespace Test
+{
+    public class PcapPacket
+    {
+        static int PACKET_HEADER_LENGTH = 16;
+        public Packet Packet;
```

Comment:
Add { get; }

Reply:
Can't, it's being set from the static method

Reply:
Ah, yes, you can do {get; private set;}